packages/react/src/floating-ui-react/hooks/useFocus.ts

```diff
@@ -194,5 +194,8 @@ export function useFocus(context: FloatingRootContext, props: UseFocusProps = {}
     [dataRef, elements.domReference, elements.triggers, onOpenChange, visibleOnly, timeout],
   );
 
-  return React.useMemo(() => (enabled ? { reference } : {}), [enabled, reference]);
+  return React.useMemo(
+    () => (enabled ? { reference, trigger: reference } : {}),
```

Comment:
What's the value of adding the trigger as key, if it's always the same as the reference value?